test/fixtures/schema/array-fixed-type.apib

```diff
@@ -0,0 +1,10 @@
+# Resource [/example]
+
+## Action [GET]
+
++ Response 200 (application/json)
+
+    + Attributes
+      - tags (array, fixed-type)
+          - (string)
```

Comment:
Please use the same example as https://github.com/apiaryio/drafter/blob/master/test/fixtures/schema/array-fixed.apib except for the `fixed-type` here.